### ISSUE TITLE
adb: inaccessible or not found on Samsung device.

### ISSUE THREAD

**Issue by Reinified** [KhunHtetzNaing/ADB-OTG#27]
Host phone for ADB: Samsung A10e
Phone im attempting to connect to: Samsung A20

Both devices run Android 11 and when I attempt to run any adb command an error occurs.

![Screenshot_20220408-135025_ADBOTG](https://user-images.githubusercontent.com/54984650/162495540-fe1656b1-ed7f-42fb-b4da-ff511adab7e1.jpg)

**Comment by TiffanyBrevier** [comment]
Has nobody replied to this yet?

**Comment by heremiah81** [comment]
Don't type "adb" before commands, just type command. eg. Do "reboot bootloader" vs "adb reboot bootloader"

**Comment by heremiah81** [comment]
Request close issue as solution was just presented

**Comment by Murffyp** [comment]
Nope, not the solution I've removed adb and still get "inaccessible or not found"

That being said, the program did work the other day, not sure what's changed... same host phone, same target tablet, same otg adapter, same cable...even tried reinstalling app....   🤷‍♂️

[Edited: fixed typos]

![Screenshot_20240704-182533-Annotated](https://github.com/KhunHtetzNaing/ADB-OTG/assets/174157558/0bb082f5-624f-4cd2-b88f-19deaddd7185)
